packages/web-components/src/components/ic-footer-link/ic-footer-link.css

```diff
@@ -30,140 +30,118 @@
-
-:host(.footer-link) a,
-:host(.footer-link) a ::slotted(a) {
-  display: flex;
-  align-items: flex-end;
-  font-weight: var(--ic-font-weight-bold);
-  color: inherit;
-  width: fit-content;
-  font-size: var(--ic-font-size-regular);
-}
-
-:host(.footer-link) a:link,
-:host(.footer-link) a ::slotted(a:link) {
-  color: inherit;
+:host(.link) .ic-link,
+:host(.link) ::slotted(a) {
+  color: var(--ic-hyperlink);
   text-decoration: underline;
   font-weight: var(--ic-font-weight-bold);
-}
-
-:host(.footer-link) a:link > ::slotted(svg),
-:host(.footer-link) a:link > ::slotted(img),
-:host(.footer-link) a ::slotted(a:link > ::slotted(svg)),
-:host(.footer-link) a ::slotted(a:link > ::slotted(img)) {
-  padding: var(--ic-space-xxxs);
-  pointer-events: none;
   transition: var(--ic-easing-transition-fast);
 }
 
-/*
-There's an apparent defect in Chrome that means this isn't always re-rendered if changed.
-Not an issue in Safari. Line 73-79 implement the same function but less clean.
-
-a:link > ::slotted(svg) ,
-a:link:visited > ::slotted(svg) {
-  fill: var(--ic-theme-text);
+:host(.link.dark) .ic-link.dark,
+:host(.link.dark) ::slotted(a) {
+  color: var(--ic-color-primary-text);
 }
-*/
 
-:host(.footer-link-light) ::slotted(svg) {
-  fill: var(--ic-architectural-white);
+:host(.link.light) .ic-link.light,
+:host(.link.light) ::slotted(a) {
+  color: var(--ic-color-white-text);
 }
 
-:host(.footer-link-dark) ::slotted(svg) {
-  fill: var(--ic-architectural-black);
+:host(.link) .ic-link:visited,
+:host(.link) ::slotted(a:visited) {
+  color: var(--ic-hyperlink-visited);
 }
 
-:host(.footer-link) a:link:hover,
-:host(.footer-link) a:link:focus,
-:host(.footer-link) a ::slotted(a:link:hover),
-:host(.footer-link) a ::slotted(a:link:focus) {
+:host(.link) .ic-link:hover,
+:host(.link) .ic-link:focus,
+:host(.link) ::slotted(a:hover),
+:host(.link) ::slotted(a:focus) {
+  outline: none;
   border-bottom: 0.25rem solid !important;
   margin-bottom: -0.25rem !important;
   text-decoration: none;
 }
 
-@supports (text-underline-offset: 25%) {
-  :host(.footer-link) a:link:hover,
-  :host(.footer-link) a:link:focus,
-  :host(.footer-link) a ::slotted(a:link:hover),
-  :host(.footer-link) a ::slotted(a:link:focus) {
+@supports (text-underline-offset: 10%) {
+  :host(.link) .ic-link:hover,
+  :host(.link) .ic-link:focus,
+  :host(.link) ::slotted(a:hover),
+  :host(.link) ::slotted(a:focus) {
     text-decoration-line: underline;
     text-decoration-thickness: 25%;
-    text-underline-offset: 25%;
+    text-underline-offset: 10%;
     border-bottom: 0 !important;
     margin-bottom: 0 !important;
   }
 }
 
-:host(.footer-link) a:link:hover,
-:host(.footer-link) a ::slotted(a:link:hover) {
-  outline: none;
+:host(.link) .ic-link:active,
+:host(.link) .ic-link:focus:active,
+:host(.link) .ic-link:visited:active,
+:host(.link) ::slotted(a:active),
+:host(.link) ::slotted(a:focus:active),
+:host(.link) ::slotted(a:visited:active) {
+  text-decoration: none;
```

Comment:
The CSS classes have been changed to `.link` but the host still has `.footer-link`. Is there supposed to be a change in host CSS classes?

Reply:
I can see there is a `.link` class on `ic-link` if it helps...

Reply:
Looks like this CSS was redundant anyways - as you say, the link class on ic-link handles all of this :)